rmarkdown.Rmd

```diff
@@ -157,7 +248,31 @@ model <- lm(Sepal.Length ~ Sepal.Width, data = iris)
 coef(model)
 ```
 
-You can use inline R code too, to insert numbers directly into the document.
+
+#### Exercise 3: adding plots and tables to your results section
+
+Time: 10 min
+
+You now need to create a chunck of code in the **Results section**. If you have any plot from your own research that you want to create feel free to do it so. Otherwise you can just use the iris data:
+
+1. Plot Sepal.length versus Sepal.Width. Include a caption and choose 6 X 10 dimensions.  
```

Comment:
We'll have to add an exercise for them to get familiar with the dataset first. And to let them know where the dataset came from.